src/main/settings/autogenerated_settings.cpp

```diff
@@ -433,6 +433,22 @@ void DefaultOrderSetting::ResetGlobal(DatabaseInstance *db, DBConfig &config) {
 	config.options.default_order_type = DBConfig().options.default_order_type;
 }
 
+//===----------------------------------------------------------------------===//
+// Disable Database Invalidation
+//===----------------------------------------------------------------------===//
+void DisableDatabaseInvalidationSetting::SetGlobal(DatabaseInstance *db, DBConfig &config, const Value &input) {
+	config.options.disable_database_invalidation = input.GetValue<bool>();
```

Comment:
Maybe similar to `AllowUnsignedExtensionsSetting` we can disallow setting this while the database is running (i.e. this needs to be set on start-up)?